src/services/storage-options/restApiService.js

```diff
@@ -2,11 +2,13 @@ import axios from "axios";
 import dayjs from "dayjs";
 import StorageInterface from "../storageInterface";
 import store from "@/store";
+import YattIntegrationHelpers from "@/integrations/YattIntegrationHelpers";
 
 export default class RestApiService extends StorageInterface {
   async getState(executionId) {
     const { data } = await axios.get(
-      `http://localhost:5000/v1/pinata/executions/${executionId}`
+      `http://localhost:5000/v1/pinata/executions/${executionId}`,
+      { withCredentials: true }
```

Comment:
You can't hardcode this here because they won't _always_ be using cookie auth.  They can still use a bearer token as well.